test/gtest/uct/ib/test_srd.cc

```diff
@@ -13,6 +13,75 @@ class test_srd : public uct_test {
     virtual void init();
 
 protected:
+    struct completion : uct_completion_t {
```

Comment:
are ucp/uct tests running over SRD now?
shall we add GET_BCOPY/GET_ZCOPY capabilities?
IMO it's more important to pass existing uct/ucp tests that to add (and keep/maintain) new srd-specific tests